### PR TITLE
Cross az attach

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: chef/cookbooks/nova/templates/default/nova.conf.erb [hunk 1]
```diff
@@ -3388,7 +3388,8 @@ insecure = <%= @cinder_insecure ? 'True' : 'False' %>
 # Cinder (or allow_availability_zone_fallback=False in cinder.conf), the volume
 # create request will fail and the instance will fail the build request.
 # (boolean value)
-#cross_az_attach = true
+#cross_az_attach >= true
```
Comment (aplanas) [inline chef/cookbooks/nova/templates/default/nova.conf.erb]: > ?

File: chef/data_bags/crowbar/template-nova.json [hunk 2]
```diff
@@ -138,7 +139,7 @@
     "nova": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 108,
+      "schema-revision": 110,
```
Comment (aplanas) [inline chef/data_bags/crowbar/template-nova.json]: the number match with the migration, is ok to jump from 108 to 110?

File: chef/cookbooks/nova/templates/default/nova.conf.erb (hunk 1 above)
Comment (toabctl) [inline chef/cookbooks/nova/templates/default/nova.conf.erb]: fixed

**Comment by jsuchome** [comment]
Don't we want to have an UI element for that?

**Comment by toabctl** [comment]
> Don't we want to have an UI element for that?

I don't think so.

**Pull request merged**